Sources/TidepoolKit/Model/TInvite.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TInvite.swift
+//  TidepoolKit
+//
+//  Created by Pete Schwamb on 5/11/23.
+//
+
+import Foundation
+
+public struct TPermissionFlag: Codable {
+    public init() {}
+}
+
+public struct TInvitePermissions: Codable {
+    let note: TPermissionFlag?
+    let upload: TPermissionFlag?
+    let view: TPermissionFlag?
+
+    public init(note: TPermissionFlag? = nil, upload: TPermissionFlag? = nil, view: TPermissionFlag? = nil) {
+        self.note = note
+        self.upload = upload
+        self.view = view
+    }
+}
+
+public struct TInviteRequest: Codable {
+    let email: String
+    let permissions: TInvitePermissions
+
+    public init(email: String, permissions: TInvitePermissions) {
+        self.email = email
+        self.permissions = permissions
+    }
+}
+
+public struct TInvite: Codable {
+    let key: String
+    let type: String
+    let email: String
+    let creatorId: String
+    let created: Date
+    let modified: Date
```

Comment:
Nit: maybe these can be named `createdAt` and `modifiedAt` to be a bit more explicit?

Reply:
We match the names defined in the API.